geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/RedisList.java

```diff
@@ -126,6 +127,25 @@ private int getArrayIndex(int listIndex) {
     return listIndex;
   }
 
+  /**
+   * @param elementToInsert element to insert into the set
+   * @param referenceElement element to insert next to
+   * @param before true if inserting before reference element, false if it is after
+   * @param region the region this instance is store in
+   * @param key the name of the list to add to
+   * @return the number of elements in the list after the element is inserted
```

Comment:
It would be good to include a mention in this comment that this method can also return -1 if the pivot element was not found.